docs/en/unloading/unload_using_insert_into_files.md

```diff
@@ -76,6 +76,7 @@ INSERT INTO
 FILES(
     "path" = "s3://mybucket/unload/data1",
     "format" = "parquet",
+    "parquet.version" = "2.6",
```

Comment:
one example for parquet.verson = 1.0 or 2.4 is enough.
the default is 2.6, most of users don't need to care about it.